web/src/stream_create_subscribers.ts

```diff
@@ -9,10 +9,8 @@ import type {ListWidget as ListWidgetType} from "./list_widget";
 import * as people from "./people";
 import {current_user} from "./state_data";
 import * as stream_create_subscribers_data from "./stream_create_subscribers_data";
-import type {CombinedPillContainer} from "./typeahead_helper";
 import * as user_sort from "./user_sort";
 
-let pill_widget: CombinedPillContainer;
```

Comment:
Not sure if this removal should have been done or not. While on one hand, this can be useful to keep around for future changes, it is also very easy to reintroduce whenever needed

Reply:
Yeah I might have left it, but it seems basically harmless to remove.